bossoidc/admin.py

```diff
@@ -13,22 +13,31 @@
 # limitations under the License.
 
 from django.contrib import admin
+from django.contrib.admin.sites import NotRegistered
+from django.contrib.auth import get_user_model
 from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
-from django.contrib.auth.models import User
 
 from bossoidc.models import Keycloak
 
+
 # Define an inline admin descriptor for Keycloak model
 # which acts a bit like a singleton
 class KeycloakInline(admin.StackedInline):
     model = Keycloak
     can_delete = False
     verbose_name_plural = 'Keycloak'
 
+
 # Define a new User admin
 class UserAdmin(BaseUserAdmin):
-    inlines = (KeycloakInline, )
+    inlines = (KeycloakInline,)
+
 
 # Re-register UserAdmin
-admin.site.unregister(User)
-admin.site.register(User, UserAdmin)
\ No newline at end of file
+User = get_user_model()
+try:
+    admin.site.unregister(User)
+except NotRegistered:
+    pass
+finally:
```

Comment:
Should `register` always be called, even when other exceptions are raised? If moved outside the `finally` block it will only run if `unregister` was successful or `NotRegistered` was raised.

Reply:
I think it could be put outside the `finally` block as well, but this will not change much.